internal/cli/server/chains/chain_test.go

```diff
@@ -24,8 +27,13 @@ func TestChain_ImportFromFile(t *testing.T) {
 			wantErr: true,
 		},
 	}
+
 	for _, tt := range tests {
+		tt := tt
```

Comment:
?

Reply:
Creating a local variable for concurrent execution, otherwise we'll get wrong variable inside goroutines.

Reply:
This is probably because we are accessing `tt` in a closure, see an explanation here: https://stackoverflow.com/questions/10116507/go-transfer-var-into-anonymous-function.

I also encountered this issue [here](https://github.com/maticnetwork/bor/blob/v0.3.0-dev/internal/cli/command.go#L34-L38) in bor.

Reply:
In closure in goroutine, yes.